design/README.md

```diff
@@ -0,0 +1,28 @@
+The budjen appliction needs to print a list of transactions in chronological order.
+
+Each class in the budjen application should one have one purpose (shown as =) and possess certain attributes (shown as -).
```

Comment:
Replace the last bit with with `(shown as bulletpoints)`.
This file is markdown so newlines with ` - ` show up as bulletpoints. In this sentence however the `-` shows up normally. Bullet points are good don't get me wrong - keep those - but the intent of this sentence is lost.